README.md

```diff
@@ -1,33 +1,41 @@
 Tax Calculator
-=======
-The Tax Calculator is a calculator for the US federal individual income tax system. In conjunction with micro data that represents US taxpayer characteristics and a set of behavioral assumptions, we use the Tax Calculator to conduct revenue estimation and distributional analyses of tax policies. 
+==============
 
-Documentation for the Tax calculator is available at http://taxcalc.readthedocs.org/en/latest/.
+The Tax Calculator simulates the US federal individual income tax
+system.  In conjunction with micro data that represent the US
+population and a set of behavioral assumptions, the Tax Calculator can
+be used to conduct revenue scoring and distributional analyses of tax
+policies.  The Tax Calculator is written in Python, an interpreted
+language that can execute on Windows, Mac, or Linux.
 
 About OSPC
-=======
-The Open-Source Policy Center (OSPC) seeks to make policy analysis more transparent, trustworthy, and collaborative by harnessing open-source methods to build cutting-edge economic models. 
-
-Disclaimer
-========
-The Tax-Calculator is currently under development. Therefore, there is NO GUARANTEE OF ACCURACY. THE CODE SHOULD NOT CURRENTLY BE USED FOR PUBLICATIONS, JOURNAL ARTICLES, OR RESEARCH PURPOSES. DO NOT CITE. Furthermore, users should be forewarned that the interface could change significantly, and the implementation is subject to wild change. The code is currently available for testing purposes only.
+==========
 
+The Open Source Policy Center (OSPC) seeks to make policy analysis
+more transparent, trustworthy, and collaborative by harnessing
+open-source methods to build cutting-edge economic models.
 
-Installation
```

Comment:
Moving this discussion over to #290. I probably should have suggested posting the motivation to github to begin with!